source/VirtualDesktop/VirtualDesktop.cs

```diff
@@ -86,5 +86,35 @@ public VirtualDesktop GetRight()
 				return null;
 			}
 		}
+
+#region IDisposable
+		private bool disposed = false;
+
+		/// <summary>
+		/// Disposes of this <see cref="VirtualDesktop"/>.
+		/// </summary>
+		/// <param name="disposeOfManagedObjects">If <see langword="true"/>, disposes of managed objects.</param>
+		protected virtual void Dispose(bool disposeOfManagedObjects)
+		{
+			if(!disposed)

```

Comment:
```suggestion
			if (!this._disposed)
```

Put a space after `if`, and use `this` for member field.